### PR TITLE
metabase.cc

### PR DESCRIPTION
**Pull request by 18dew**
Metabase.cc

### CODE REVIEW

**Comment by 409H** [comment]
Any relation to `metabase.com`? It was hosted on the same server

https://urlscan.io/ip/54.194.41.141

**Comment by 18dew** [comment]
Metabase.com is a different project , metabase.one and metabase.cc are related to same project.

**Comment by 18dew** [comment]
Added metabase.network.

File: src/config.json [hunk 1]
```diff
@@ -212,7 +212,11 @@
     "cryptokitties.co",
     "remme.io",
     "jibrel.network",
-    "twinity.com"
+    "twinity.com",
+    "www.metabase.cc",
```
Comment (kumavis) [inline src/config.json]: we block everything under the specified domain so
`"www.metabase.network"`
is blocked by 
`"metabase.network"`

**Comment by kumavis** [comment]
sry took so long

**Pull request merged**